src/libertem/executor/pipelined.py

```diff
@@ -666,14 +666,22 @@ def _start_pool(self) -> WorkerPool:
             # FIXME: check process liveness here, too, to catch early startup
             # failures without having to run into the timeout
```

Comment:
This comment is more relevant now that the error message is explicit. The user could be experiencing process startup errors, i.e. the process is dead by the time we wait for its message, and the message would lead them down the wrong track ? We could do our best and raise a different error `if not pool.workers[i].process.is_alive()` ? It might not catch everything but it's a start.

Reply:
Good point - I'll try to take a stab at this.